### PR TITLE
Add load data

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1311

- Add procedures for how to load data from client library or Telegraf plugin via the UI
- Update intro on Write data page to include load data, best practices, and bulk ingest

Note, while the original issue mentions adding the following sections to the UI:
Client libraries
Telegraf plugins
Flux sources
Integrations
Developer tools
Demo data (cloud-only)

....only the client libraries and Telegraf plugins have been added thus far

### CODE REVIEW

File: content/influxdb/v2.0/write-data/_index.md [hunk 1]
```diff
@@ -163,15 +166,41 @@ If you missed the Quick Start option, you can [manually create a scraper](/influ
...
+### Load data from a Telegraf plugin in the UI
+
+1. In the navigation menu on the left, click **Data (Load Data)** > **Sources**.
+    {{< nav-icon "data" >}}
+2. Do one of the following:
+   - Enter a specific Telegraf plugin to search for in the **Search data writing methods** field.
+   - Scroll down to browse available plugins.
+3. Click the plugin to load data from.
+4. [Install Telegraf](/telegraf/v1.15/introduction/installation/).
+5. Copy the default configuration script in the UI, and then add the script to your [Telegraf configurations](/influxdb/v2.0/telegraf-configs/).
```
Comment (sanderson) [inline content/influxdb/v2.0/write-data/_index.md:202]: This should probably link to information about the Telegraf configuration file (https://docs.influxdata.com/telegraf/v1.15/introduction/getting-started/#configure-telegraf). We could then add an optional step about adding the Telegraf configuration to InfluxDB (this is what /influxdb/v2.0/telegraf-configs/ walks through).

**Review by noramullen1 — APPROVED** [review]

File: content/influxdb/v2.0/write-data/load-data.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: Load data from sources in the InfluxDB user interface (UI)
+seotitle: Load data source in UI
+list_title: Load data source in UI
+weight: 105
+description: >
+  Load data source in UI
```
Comment (sanderson) [inline content/influxdb/v2.0/write-data/load-data.md:7]: This is the summary that will appear in search engine results. We should expand the description to include popular data sources.

File: content/influxdb/v2.0/write-data/load-data.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+### Load data from a Telegraf plugin in the UI
+
+1. In the navigation menu on the left, click **Data (Load Data)** > **Sources**.
+    {{< nav-icon "data" >}}
+2. Do one of the following:
+   - Enter a specific Telegraf plugin to search for in the **Search data writing methods** field.
+   - Scroll down to browse available plugins.
+3. Click the plugin to load data from.
+4. [Install Telegraf](/telegraf/v1.15/introduction/installation/).
+5. Copy the default configuration script in the UI, and then add the script to [Configure Telegraf](https://docs.influxdata.com/telegraf/v1.15/introduction/getting-started/#configure-telegraf).
```
Comment (sanderson) [inline content/influxdb/v2.0/write-data/load-data.md:43]: This should be a relative link, not an absolute link:

```diff
- https://docs.influxdata.com/telegraf/v1.15/introduction/getting-started/#configure-telegraf
+ /telegraf/v1.15/introduction/getting-started/#configure-telegraf
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Just some minor tweaks.

**Pull request merged**